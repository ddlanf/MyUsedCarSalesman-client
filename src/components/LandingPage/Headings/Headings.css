.heading-box{
  margin-top: 1vw;
  margin-bottom: 3vw;
  width: 80%;
  margin-left: auto;
  margin-right: auto;
  text-align: center;
  display: block;
}

.heading-image{
  display: block;
  margin-right: auto;
  margin-left: auto;
  margin-top: 0;
  margin-bottom: 0;
  max-height: 180px;
  height: 18vw;
}

.cash{
  margin-top: -3%;
}

.header-main{
  font-size: 5.5vw;
  width: 95%;
  padding-top: 2px;
  margin-left: auto;
  margin-right: auto;
}

.sub-text{
  text-align: left;
  margin-left: auto;
  margin-right: auto;
  width: 80%;
}

.sub-text > p{
  font-size: 4.3vw;
  line-height: normal;
  text-align: center;
}

.new-para{
  display: block;
  margin-bottom: 2%;
  padding: 0;
}

.landingpage-break{
  display: block;
  margin: 0;
}


/*for demo purposes only*/
.demo > h1{
  font-size: 5vw;
  margin: 1%;
  cursor: pointer;
}

.demo-break{
  display: block;
}

.demo{
  width: 80%;
  border: solid black 1px;
  top: 4%;
  right: 2%;
  background-color:  #EAEDEC;
  text-align: center;
  margin-left: auto;
  margin-right: auto;
  font-size: 4vw;
  line-height: 4vw;
  margin-top: 5%;
  line-height: 6vw;
}

ol{
  text-align: left;
  margin-left: 3%;
}


@media screen and (min-width: 1000px) {

  .demo > h1{
    font-size: 40px;
    margin: 0;
  }

  .demo{
    font-size: 30px;
  }
  /*ends here*/

  .header-main{
    font-size: 50px;
  }
    
  .sub-text > p{
    font-size: 30px;
  }

  .cash{
    margin-top: -30px;
  }
}