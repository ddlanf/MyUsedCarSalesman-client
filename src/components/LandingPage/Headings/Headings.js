import React, { Component } from 'react'
import './Headings.css'

export default class Headings extends Component {
    render() {
        return (
            <>
               <section>
                    <div className="heading-box">
                        <h1 className="header-main">
                            The most effortless way to sell your vehicle, 
                            while maximizing your return
                        </h1>
                    </div>
                    <img 
                        className="heading-image money-and-car-image"
                        src={require('../../../Utils/myusecarsalesman_pics/messageImage_1585474997595.png')}
                        alt="money and car"
                    />
                    <div className="sub-text">
                        <p>
                            MyUsedCarSalesman helps you find an individual who is willing to do
                            all the complicated and tedious process of selling your old car. 
                        </p>
                    </div>
                </section>

                <section>
                    <div className="heading-box">
                        <h1 className="header-main">Forget the trade-in value!</h1>
                    </div>
                    <div className="sub-text">
                        <p>
                            Dealers will take your car off your hand for a lot less money than what your car is 
                            actually worth so that they get the most profit out of your vehicle.
                        </p>
                        <img 
                        className="heading-image dealer"
                        src={require('../../../Utils/myusecarsalesman_pics/presentation.png')}
                        alt="dealer"
                        />
                        <p>
                        In MyUsedCarSalesman, you decide how much value you will get out your vehicle.
                        </p>
                    </div>
                </section>

                <section>
                        <div className="heading-box">
                            <h1 className="header-main">Looking to earn some cash? <span className="landingpage-break">Become a seller!!</span> </h1>
                        </div>
                        <img 
                        className="heading-image cash"
                        src={require('../../../Utils/myusecarsalesman_pics/CartoonMoney.png')}
                        alt="money"
                        />
                        <div className="sub-text">
                            <p>
                                <span className="new-para">
                                Anyone can become a seller on MyUsedCarSalesman! Once you have an account, simply accept any contracts posted by other users and get sellin'! 
                                </span>
                                No diploma, certification, nor prior sales experience necessary, let your results speak for itself.
                            </p>
                        </div>
                </section> 
            </>
        )
    }
}
