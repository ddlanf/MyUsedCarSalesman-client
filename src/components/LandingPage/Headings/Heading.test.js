import React from 'react';
import ReactDOM from 'react-dom';
import Headings from './Headings';
import renderer from 'react-test-renderer';

describe('Headings component', () => {
    
    it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Headings/>, div);
    ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {
        const tree = renderer
          .create(<Headings/>)
          .toJSON();
        expect(tree).toMatchSnapshot();  
      });
});