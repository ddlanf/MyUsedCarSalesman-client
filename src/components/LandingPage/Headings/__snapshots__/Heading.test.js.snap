// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headings component renders the UI as expected 1`] = `
Array [
  <div
    className="demo"
  >
    <h1
      onClick={[Function]}
    >
      Click here for instruction
    </h1>
    
  </div>,
  <section>
    <div
      className="heading-box"
    >
      <h1
        className="header-main"
      >
        The most effortless way to sell your vehicle, while maximizing your return
      </h1>
    </div>
    <img
      alt="money and car"
      className="heading-image money-and-car-image"
      src="moneyandcar.png"
    />
    <div
      className="sub-text"
    >
      <p>
        MyUsedCarSalesman helps you find an individual who is willing to do all the complicated and tedious process of selling your old car.
      </p>
    </div>
  </section>,
  <section>
    <div
      className="heading-box"
    >
      <h1
        className="header-main"
      >
        Forget the trade-in value!
      </h1>
    </div>
    <div
      className="sub-text"
    >
      <p>
        Dealers will take your car off your hand for a lot less money than what your car is actually worth so that they get the most profit out of your vehicle.
      </p>
      <img
        alt="dealer"
        className="heading-image dealer"
        src="dealer.png"
      />
      <p>
        In MyUsedCarSalesman, you decide how much value you will get out your vehicle.
      </p>
    </div>
  </section>,
  <section>
    <div
      className="heading-box"
    >
      <h1
        className="header-main"
      >
        Looking to earn some cash? 
        <span
          className="landingpage-break"
        >
          Become a seller!!
        </span>
         
      </h1>
    </div>
    <img
      alt="money"
      className="heading-image cash"
      src="cash.png"
    />
    <div
      className="sub-text"
    >
      <p>
        <span
          className="new-para"
        >
          Anyone can become a seller on MyUsedCarSalesman! Once you have an account, simply accept any contracts posted by other users and get sellin'!
        </span>
        No diploma, certification, nor prior sales experience necessary, let your results speak for itself.
      </p>
    </div>
  </section>,
]
`;
